src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Table(name = "books")
+@Data
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NotNull
+    private String title;
+    @NotNull
+    private String author;
+    @Column(unique = true, nullable = false)
+    private String isbn;
+    @NotNull
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
```

Comment:
It might be better to indicate the name of the column in the DB for this field?

Reply:
By default, all columns receive field names. And in case when field has name like "coverImage" -  column gets a name "cover_image".  So all column have correct name =)  One thing that worries me - is why are they in random order? 
![table_name](https://github.com/ystankevych/book-app/assets/105653924/e17bd39c-6082-4f0a-b2d8-216ab01d9206)